src/layout/navigation/NavItem.tsx

```diff
@@ -6,9 +6,11 @@ import React, {
   Dispatch,
   RefObject,
 } from 'react'
+import Link from 'gatsby-link'
```

Comment:
@SonyaMoisset this will need the lgtm comment

Reply:
@egmcdonald Thanks 👍